### DISCUSSION TITLE
P editor not show value

### DISCUSSION THREAD

**Discussion by duongduykhuong** [primefaces/community#1716] [PrimeNG]
Please help me! 
P editor cannot show value Text to input editor. I used OMEGA template
![1](https://github.com/primefaces/community/assets/167502213/68ce5b8c-5e06-4e6e-bd0a-f6c0b4dff30e)
![2](https://github.com/primefaces/community/assets/167502213/4b66e8f1-59ee-408a-9cf3-8fb0e0b114be)
![3](https://github.com/primefaces/community/assets/167502213/9a886fb8-6031-47c9-9a9a-857f05e7a0af)

**Comment by cetincakiroglu** [comment]
Hi,

I couldn't replicate the issue with the latest version of PrimeNG (17.14.1). https://stackblitz.com/edit/hcqxvh?file=src%2Fapp%2Fdemo%2Feditor-basic-demo.ts

Which version are you using and could you please reproduce the issue in a stackblitz example and share us so we can better understand what's wrong?

**Comment by Emiliorth** [comment]
Hi, I tested on 17.13 and 17.14.1 and also p-editor is not showing value
I use p-editor with formControlName and with patchValue with value:` '<div>Bla</div>'` as string
Control itself contains new value, but p-editor visually doesn't show it. When I input something in it, it overwrites control's value and ofcourse then shows my typed value;

**Reply by ThoSap** [reply]
@Emiliorth which quill version do you have installed, `1.3.7` or `2.0.0`?

**Reply by cetincakiroglu** [reply]
@Emiliorth Could you please share a stackblitz example, so we can better understand the issue?

**Reply by Emiliorth** [reply]
@cetincakiroglu It is a simple one control FormGroup with it's function to patch control's value.
p-editor doesn't show this value

@ThoSap 2.0.0. Installed 1.3.7 and worked! :)

**Reply by ThoSap** [reply]
@Emiliorth 2.0.0 is currently not supported by the PrimeNG Editor as these code changes are necessary
https://github.com/primefaces/primeng/issues/14721#issuecomment-2068582938

So if 1.3.7 works in the meantime you can mark this as accepted.

**Reply by ThoSap** [reply]
@Emiliorth you can now update to https://github.com/primefaces/primeng/releases/tag/17.18.3 and Quill 2.0.2 😉

**Comment by esoftware-github** [comment]
I have the same issue and can't find any documentation on how to solve it.

**Reply by Yassinmoh** [reply]
Hi, did you find any solution to this problem?

**Reply by Yassinmoh** [reply]
Hello I use this solution, it works with me :
https://github.com/primefaces/primeng/issues/14721#issuecomment-2092910922

**Reply by ThoSap** [reply]
Hi @Yassinmoh,
good that my workaround works for you 😋

Since PrimeNG https://github.com/primefaces/primeng/releases/tag/17.18.3 you can also use Quill `2.X.X` (last release as of today 2.0.2) so you no longer need this workaround.

**Reply by Yassinmoh** [reply]
Hi @ThoSap 
I appreciate your help and the workaround made things much smoother. I’ll definitely update to the latest PrimeNG version
